telcell/celldb/pgdatabase.py

```diff
@@ -0,0 +1,414 @@
+import csv
+import datetime
+import math
+import warnings
+from typing import Optional, List, Iterable, Callable, Tuple
+
+import geopy
+
+from telcell.cell_identity import (
+    Radio,
+    CellIdentity,
+    CellGlobalIdentity,
+    EutranCellGlobalIdentity,
+)
+from . import duplicate_policy
+from .cell_collection import CellCollection, Properties
+from ..data.models import RD_TO_WGS84, WGS84_TO_RD
+from ..geography import Angle
+
+RD_X_RANGE = (7000, 300000)
```

Comment:
in models.py staan ook een rd_x en y_range. Wel met andere nummers. Welke is correct, en eentje kiezen en de andere weghalen